### PR TITLE
Feature/0 0 8

### PR DESCRIPTION
**Pull request by hexthedev**
### Summary 
Goes with the new 0.0.8 database and scene

### CODE REVIEW

**Comment by Ysabz** [comment]
Added these to this PR : 
1) fixed the bug with parsing ( data is splitted by ":" which causes issue if the name contains ":") 
2) Added the functionality to only show enabled inputs (enabled = true if this arg is not set in json)
3) Added the functionality to parse boolean inputs and send commands
4) Added the functionality to pause the animations at the start and end of presentation and when exit from playmode

File: src/components/PreviewOverlay.tsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ function PreviewOverlay(): JSX.Element {
           onClick={() => {
             toggleOverlay();
             defaultCameraView();
+            pauseApparatus();
```
Comment (mckenzietsean) [inline src/components/PreviewOverlay.tsx:66]: Function pauseApparatus() should also be used in Overlay.tsx for when the user presses the ESC key while previewing.

Comment (mckenzietsean) [inline src/components/PreviewOverlay.tsx:66]: Function pauseApparatus() should also be used in Overlay.tsx for when the user presses the ESC key while previewing.

**Review by mckenzietsean — CHANGES_REQUESTED** [review]

File: src/components/PreviewOverlay.tsx (hunk 1 above)
Comment (Ysabz) [reply src/components/PreviewOverlay.tsx:66]: fixed

**Review by mckenzietsean — APPROVED** [review]
Everything looks good now.

**Pull request merged**